lohica-js-network/src/components/RegForm.js

```diff
@@ -1,162 +1,123 @@
 import React from 'react';
-import {FormControl, FormGroup, ControlLabel, Form, Col, Row, Grid, Button, Radio} from 'react-bootstrap'
-import RegistrationInput from './RegFormInput';
+import {Form, Row, Grid, Button} from 'react-bootstrap';
+import {connect} from 'react-redux';
+import NameInput from './inputComponents/NameInput';
```

Comment:
You may create an additional file inside ./inputComponents folder. Call it index.js. This file would export an object with NameInput, EmailInput etc
Then using destructurisation you would import all needed components in one line